docs/composition/file_patterns.md

```diff
@@ -207,8 +207,8 @@ pattern[index]
 ## From file pattern to `PCollection`
 
 As covered in {doc}`index`, a recipe is composed of a sequence of Apache Beam transforms.
-The data collection that Apache Beam transforms operates on is a
-[`PCollection`](https://beam.apache.org/documentation/programming-guide/#pcollections).
+The data Apache Beam transforms operate on are
+[`PCollections`](https://beam.apache.org/documentation/programming-guide/#pcollections).
```

Comment:
So do the iterables of a filepattern become pcollections? This is a bit unclear to me.

Reply:
Thanks for digging into this, @abarciauskas-bgse!

Yes, this is correct. A FilePattern is iterated over (via [`.items`](https://github.com/pangeo-forge/pangeo-forge-recipes/blob/6e40279b7f5740600b77a348dd48d09f11467564/pangeo_forge_recipes/patterns.py#L218-L221), not `__iter__`), and this is converted into a PCollection:

```python

pattern = FilePattern(...)

recipe = beam.Create(pattern.items())
```

```